lua/nvim-treesitter/parsers.lua

```diff
@@ -17,16 +17,12 @@ local list = setmetatable({}, {
 
     rawset(table, parsername, setmetatable(parserconfig, {
       __newindex = function(parserconfigtable, key, value)
-        if key == "used_by" then
-          ft_to_parsername[value] = parsername
```

Comment:
why is `used_by` not needed anymore?

Reply:
i believe used_by is already set after the rawset call, removing this code did not affect highlighting in jsx files so i believed it was ok to remove it.

Reply:
It is set in the update_ft_to_parser name that is called two lines after the removed code.